fractal/server/app/api/v1/task.py

```diff
@@ -100,6 +103,34 @@ async def create_task(
     return db_task
 
 
+@router.patch("/{task_id}/", response_model=TaskRead)
+async def patch_task(
+    task_id: int,
+    task_update: TaskUpdate,
+    user: User = Depends(current_active_user),
+    db: AsyncSession = Depends(get_db),
+):
+
+    # FIXME add user-owned tasks
+
+    db_task = await db.get(Task, task_id)
+
+    for key, value in task_update.dict(exclude_unset=True).items():
+        debug(key, value)
```

Comment:
remove

Reply:
I guess the only thing to remove is the `debug` call.
Done.